components/eamxx/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -348,6 +348,13 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
   void add_field (const FieldIdentifier& fid, const std::list<std::string>& groups, const int ps)
   { add_field<RT>(FieldRequest(fid,groups,ps)); }
 
+  // Specialization for add_field to tracer group
+  template<RequestType RT>
+  void add_tracer (const std::string& name, const FieldLayout& layout,
```

Comment:
What if we passed the grid obj rather than a layout and a grid name? The impl could grab them from the grid.